src/ucp/wireup/wireup.c

```diff
@@ -209,6 +216,76 @@ static void ucp_wireup_remote_connected(ucp_ep_h ep)
     }
 }
 
+static unsigned ucp_wireup_listener_accept_cb_progress(void *arg)
+{
+    ucp_ep_h ep = arg;
+
+    ep->flags |= UCP_EP_FLAG_USED;
+    ucp_ep_ext_gen(ep)->listener->cb(ep, ucp_ep_ext_gen(ep)->listener->arg);
+
+    return 0;
+}
+
+int ucp_wireup_listener_accept_cb_remove_filter(const ucs_callbackq_elem_t *elem,
+                                                void *arg)
+{
+    ucp_ep_h ep = elem->arg;
+
+    return (elem->cb == ucp_wireup_listener_accept_cb_progress) && (ep == arg);
+}
+
+static void ucp_wireup_schedule_listener_cb(ucp_ep_h ep)
```

Comment:
the 3 functions ucp_wireup_listener_accept_cb_progress, ucp_wireup_listener_accept_cb_remove_filter and ucp_wireup_schedule_listener_cb should be in ucp_listener.c